cmd/renterd/logger.go

```diff
@@ -57,10 +57,10 @@ func NewLogger(dir, filename string, cfg config.Log) (*zap.Logger, func(context.
 
 		var encoder zapcore.Encoder
 		switch cfg.File.Format {
-		case "human":
-			encoder = humanEncoder(false) // disable colors in file log
-		default: // log file defaults to json
+		case "json":
 			encoder = jsonEncoder()
+		default: // log file defaults to 'human'
```

Comment:
I guess it's overkill but shouldn't we add a migration to backup the renter.log file and start fresh using this new log file format.

Reply:
I don't think there is a lot of value in that. Writing a migration probably takes longer than just fixing the log manually in case you missed that change in the changelog.